### PR TITLE
Clarified port forwarding and auth for GCP

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1082

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1083)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
Preview:
* https://deploy-preview-1083--competent-brattain-de2d6d.netlify.com/docs/other-guides/accessing-uis/
* https://deploy-preview-1083--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/oauth-setup/
* https://deploy-preview-1083--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-ui/
* https://deploy-preview-1083--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/deploy-cli/

**Comment by sarahmaddox** [comment]
This PR is now ready for review. Applying a hold to allow for multiple reviewers.

/hold

/assign @jlewi 
/assign @amygdala

**Comment by sarahmaddox** [comment]
I haven't addressed the port-forwarding sections in the [GCP e2e tutorial](https://www.kubeflow.org/docs/gke/gcp-e2e/) as that tutorial is pretty broken anyway. I'll tackle port-forwarding along with other fixes in PR https://github.com/kubeflow/website/pull/1079.

**Comment by jlewi** [comment]
Thanks @sarahmaddox 

/lgtm
/approve
/hold
@amygdala any comments?

**Comment by amygdala** [comment]
that /lgtm to me.  Thank you, Sarah.

File: content/docs/other-guides/accessing-uis.md [hunk 1]
```diff
@@ -36,6 +28,15 @@ The central UI dashboard looks like this:
   alt="Kubeflow central UI"
   class="mt-3 mb-3 border border-info rounded">
 
+## Overview of accessing the Kubeflow UIs
+
+To access the Kubeflow UIs you need to connect to the
```
Comment (joeliedtke) [inline content/docs/other-guides/accessing-uis.md]: Suggestion: add a comma after UIs

To access the Kubeflow UIs, you need to connect to the

File: content/docs/other-guides/accessing-uis.md [hunk 2]
```diff
@@ -36,6 +28,15 @@ The central UI dashboard looks like this:
   alt="Kubeflow central UI"
   class="mt-3 mb-3 border border-info rounded">
 
+## Overview of accessing the Kubeflow UIs
+
+To access the Kubeflow UIs you need to connect to the 
+[ISTIO gateway](https://istio.io/docs/concepts/traffic-management/#gateways) that
```
Comment (joeliedtke) [inline content/docs/other-guides/accessing-uis.md]: Suggestion: replace "ISTIO" with "Istio"

The [Istio docs](https://istio.io/docs/concepts/)  use "Istio."

File: content/docs/other-guides/accessing-uis.md [hunk 3]
```diff
@@ -36,6 +28,15 @@ The central UI dashboard looks like this:
...
   class="mt-3 mb-3 border border-info rounded">
 
+## Overview of accessing the Kubeflow UIs
+
+To access the Kubeflow UIs you need to connect to the 
+[ISTIO gateway](https://istio.io/docs/concepts/traffic-management/#gateways) that 
+provides access to the Kubeflow 
+[service mesh](https://istio.io/docs/concepts/what-is-istio/#what-is-a-service-mesh).
+
+How you access the ISTIO gateway varies depending on how you've configured it.
```
Comment (joeliedtke) [inline content/docs/other-guides/accessing-uis.md]: Same as above ISTIO -> Istio

File: content/docs/other-guides/accessing-uis.md [hunk 4]
```diff
@@ -58,12 +59,20 @@ guide to
...
+  * You've deployed Kubeflow on GCP using the 
+    [GCP deployment UI](/docs/gke/deploy/deploy-ui/) or the default settings 
+    with the [CLI deployment](/docs/gke/deploy/deploy-cli/). (If you want to
+    use port forwarding, you must deploy Kubeflow on an existing Kubernetes 
+    cluster using the [`kfctl_k8s_istio` 
+    configuration](/docs/started/k8s/kfctl-k8s-istio/).)
 
-  * you've configured the ISTIO ingress to only accept 
-HTTPS traffic on a specific domain or IP address
+  * You've configured the ISTIO ingress to only accept
```
Comment (joeliedtke) [inline content/docs/other-guides/accessing-uis.md]: ISTIO -> Istio

File: content/docs/other-guides/accessing-uis.md [hunk 5]
```diff
@@ -58,12 +59,20 @@ guide to
...
+    cluster using the [`kfctl_k8s_istio` 
+    configuration](/docs/started/k8s/kfctl-k8s-istio/).)
 
-  * you've configured the ISTIO ingress to only accept 
-HTTPS traffic on a specific domain or IP address
+  * You've configured the ISTIO ingress to only accept 
+    HTTPS traffic on a specific domain or IP address.
 
-  * you've configured the ISTIO ingress to perform an authorization check (e.g. using IAP or Dex)
+  * You've configured the ISTIO ingress to perform an authorization check
```
Comment (joeliedtke) [inline content/docs/other-guides/accessing-uis.md]: ISTIO -> Istio

File: content/docs/other-guides/accessing-uis.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/other-guides/accessing-uis.md]: Done

File: content/docs/other-guides/accessing-uis.md (hunk 2 above)
Comment (sarahmaddox) [reply content/docs/other-guides/accessing-uis.md]: Done.

**Comment by sarahmaddox** [comment]
Thanks for the review everyone! @joeliedtke I've addressed your review comments. PTAL.

**Comment by sarahmaddox** [comment]
Wow that was super fast. Thanks Joe!

**Pull request merged**